packages/yoga/src/Popover/web/Popover.jsx

```diff
@@ -13,10 +13,26 @@ function Popover({
   height,
   zIndex,
   a11yId,
+  hover,
   ...props
 }) {
+  const ref = useRef(null);
```

Comment:
Make the component a forwardRef, so you cam also receive external refs. WDYT?
(Example in dialog)
`    const popoverRef = useCombinedRefs(forwardedRef);`